app/main/views.py

```diff
@@ -100,7 +100,7 @@ def update(service_id, section):
     # Turn responses which have multiple parts into lists
     for key in request.form:
         item_as_list = request.form.getlist(key)
-        if len(item_as_list) > 1:
+        if content.get_question(key)['type'] == 'list':
```

Comment:
This change breaks `pricing` type questions. It will also future-break series of checkboxes elements which get `POST`ed as lists.